src/System.Runtime.Numerics/src/System/Numerics/BigNumber.cs

```diff
@@ -325,6 +325,18 @@ internal static bool TryValidateParseStyleInteger(NumberStyles style, out Argume
         [SecuritySafeCritical]
         internal static bool TryParseBigInteger(string value, NumberStyles style, NumberFormatInfo info, out BigInteger result)
         {
+            if (value == null)
+            {
+                result = default(BigInteger);
+                return false;
+            }
+
+            return TryParseBigInteger(AsReadOnlySpan(value), style, info, out result);
+        }
+
+        [SecuritySafeCritical]
```

Comment:
Simply because it was on the other overload.  I figure either a) we've not yet cleaned these out and so it can be removed when the rest are, or b) someone left it on purpose and so I should have it here as well.